src/Race.ts

```diff
@@ -0,0 +1,55 @@
+import {Stat, statDict} from './Stat'
+import {Skill} from './Skill'
+
+const races = ['human', 'elf', 'half giant'] // Temporary list just to test setRace method.
+
+interface IRace {
+  // Some values are optional to allow for testing.
+  stats: Stat
+  skills?: Skill
+  name?: string
+}
+
+export default class Race implements IRace {
+  // Optional variable types for testing
+  constructor(
+    public stats: Stat,
+    public skills?: Skill,
+    public name?: string
+  ) {}
+
+  setRace(raceInput: string) {
+    // Type string for testing
+    for (var race of races) {
+      if (raceInput === race) {
+        return console.log(race)
+      }
+    }
+    if (raceInput == 'human') {
+      // This way of doing it is less than optimal because i would have to create a funciton for every race. The above is a better i think.
```

Comment:
You've hit upon the core problem of runes. :rofl: This should be defined in the data instead of in the code, but each rune should have a list of modifiers with some default.

I think you'll basically want to read json or similar data to a data object, and then iterate over all of the possible types of bonuses that you get.